edb/pgsql/metaschema.py

```diff
@@ -4409,9 +4409,54 @@ def __init__(self) -> None:
         )
 
 
+class UuidGenerateV1mcFunction(dbops.Function):
```

Comment:
Is there any chance these extra wrappers will slow things down? Probably they can get inlined?
(Is preventing that why they are marked parallel_safe?)

Reply:
They should get inlined.  See criteria for inlining [here](https://github.com/postgres/postgres/blob/919c486a275bcdd83d2add77da87b6edbd91418c/src/backend/optimizer/util/clauses.c#L4401)

>  (Is preventing that why they are marked parallel_safe?)

No, the underlying function is marked as parallel-safe, so the wrapper should also.